File: programmers/number_string_alphabet.cpp
#include <iostream>
#include <vector>
#include <string>
#include <regex>

using namespace std;


int main() {
    cin.tie(0);
    cout.tie(0);
    ios_base::sync_with_stdio(false);
    
    string s;
    cin >> s;
    
    s = regex_replace(s, regex("zero"), "0");
    s = regex_replace(s, regex("one"), "1");
    s = regex_replace(s, regex("two"), "2");
    s = regex_replace(s, regex("three"), "3");
    s = regex_replace(s, regex("four"), "4");
    s = regex_replace(s, regex("five"), "5");
    s = regex_replace(s, regex("six"), "6");
    s = regex_replace(s, regex("seven"), "7");
    s = regex_replace(s, regex("eight"), "8");
    s = regex_replace(s, regex("nine"), "9");

    cout << stoi(s);

    return 0;
}

File: programmers/number_stringalphabet.java
class Solution {
    public int solution(String s) {
		String res = s.replaceAll("zero", "0");
		res = res.replaceAll("one", "1");
		res = res.replaceAll("two", "2");
		res = res.replaceAll("three", "3");
		res = res.replaceAll("four", "4");
		res = res.replaceAll("five", "5");
		res = res.replaceAll("six", "6");
		res = res.replaceAll("seven", "7");
		res = res.replaceAll("eight", "8");
		res = res.replaceAll("nine", "9");
        
        return Integer.parseInt(res);
    }
}public class number_string&alphabet {
    
}
